src/index.ts

```diff
@@ -24,9 +24,11 @@ const updateGame = () => {
   console.clear();
   gameField.updateGameField();
   player.putTank(gameField);
-  enemy.putTank(gameField);
   player.updateProjectiles(gameField);
-  enemy.updateProjectiles(gameField);
+  enemies.forEach(enemy => {
```

Comment:
Why not use for...of instead forEach? As I know it's more reliable.

Reply:
forEach is almost the same as for or for..of , only slower. There's not much performance difference between the two loops, so we can use anything we want. I prefer forEach because its more clean code

Reply:
OK